.github/workflows/test.yml

```diff
@@ -5,9 +5,9 @@ on:
     branches: [ master ]
 
 jobs:
-  test-windows:
-    name: test-windows py3${{ matrix.python-minor }}
-    runs-on: windows-2019
+  test-linux:
+    name: test linux py3${{ matrix.python-minor }}
+    runs-on: ubuntu-16.04
```

Comment:
Didn't you want to use at least ubuntu 18?

Reply:
I can't - locally-stashed qemu doesn't work, and I couldn't find the replacement quickly.